core/src/main/java/smile/regression/RegressionTree.java

```diff
@@ -493,17 +493,18 @@ void unwind(int i) {
             double n = w[l];
             if (po != 0) {
                 for (int j = l - 1; j >= 0; j--) {
-                    double t =  n / ((j+1) * po);
+                    double t =  (n * (l + 1)) / ((j+1) * po);
```

Comment:
I refer the c++ version: https://github.com/slundberg/shap/blob/cde70feb211443cfbd9f599fcded8fe00b9d6a3d/shap/tree_shap.h#L377-L390.

Why is it wrong?

Reply:
ok... maybe the c++ version is a bit different from the python version: https://github.com/slundberg/shap/blob/master/shap/explainers/pytree.py#L291-L302